trillian_map_api.proto

```diff
@@ -200,3 +219,16 @@ service TrillianMap {
     };
   }
 }
+
+// TrillianWriteMap defines a service to allow writes against a Verifiable Map
+// that will be readable via the TrillianMap service. The write API does not
+// expose any Merkle Tree properties. This allows key/value writes to be
+// decoupled from the Merkle Tree synthesis and publishing.
+service TrillianWriteMap {
```

Comment:
`TrillianMapWrite` (or `TrillianMapUpdate`) feel a little more natural to me, but I don't feel strongly about it :)

Reply:
Along similar lines I was going to suggest something like `TrillianMapWriter`?

Reply:
SGTM.